md_text/3-3.md

```diff
@@ -4,112 +4,201 @@
 
 ## `p`要素
 
-`p`要素は段落を表します。日本語の文章の段落は、通常いくつかの文で構成される、文のかたまりと捉えることができます。その一方で、HTML仕様における段落の考え方があります。これについては後述します。
+`p`要素は「段落」(paragraph) を表します。段落は、いくつかの文で構成される、文のかたまりです。
 
-より具体的な要素が適当な場合は、専用の要素を使用するのが望ましいです。例えば、次のマークアップは技術的に正しいです。
+詳しくは後述しますが、ここでいう段落は「構造的」(structural)なまとまりであり、必ずしも「論理的」(logical)なまとまりであるとは限りません。<!--国語の授業で意味段落、形式段落という言葉を習ったかもしれませんが、形式段落に近いと考えて良いでしょう。-->
```

Comment:
日本語の段落に相当するとは限らないという難しさはあるが、

https://html.spec.whatwg.org/multipage/grouping-content.html#the-p-element

> Authors wishing to conveniently style such “logical” paragraphs consisting of multiple “structural” paragraphs can use the div element instead of the p element.

ということみたいなので、一旦ママで。